waspc/src/Wasp/Analyzer/Parser/Token.hs

```diff
@@ -2,6 +2,7 @@ module Wasp.Analyzer.Parser.Token where
 
 import Wasp.Analyzer.Parser.SourcePosition (SourcePosition, calcNextPosition)
 
+-- NOTE: If you add a new keyword, make sure to update the error message in CreateNewProject.hs.
```

Comment:
I was pretty unhappy with this so I tried to make it nicer.

The changes are in a different branch (I'll update this comment with a link as soon as it's ready). It does remove the duplication, but it might be overegineered.